src/damf.rs

```diff
@@ -678,8 +690,8 @@ fn roundtrip() {
 presentations:
   - type: home
     simplified: false
-    metadata: NaturesFury.atmos.metadata
-    audio: NaturesFury.atmos.audio
+    metadata: Natures  -  Fury.atmos.metadata
+    audio: Natures  -  Fury.atmos.audio
```

Comment:
It was great to have these unit tests already written, to demonstrate that refactoring the function didn't break the tests. The tests passed without any modification. I did, however, want to make sure the tests would exercise the refactored functionality, so I changed these test fixtures.